src/rules/maxLineLengthRule.ts

```diff
@@ -19,6 +19,8 @@ import * as ts from "typescript";
 
 import * as Lint from "../lint";
 
+import { IDisabledInterval } from "../language/rule/rule";
+
 export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
```

Comment:
Let's update this as well - this info is used to generate the docs website when a release is published